### PR TITLE
Centrifuge Recipes for U235 - Tweaks

### PR DESCRIPTION
**Pull request by Steelux8**
See https://github.com/GTNewHorizons/GT5-Unofficial/pull/1670.

I made the PR again, after the GT_MachineRecipeLoader changes. I also changed the small dust chances to normal dust chances by using the naive method (divide chances by 4), but I also doubled the chances from there from Dream's feedback saying that the recipe probably wasn't worth it still. On top of that, discussion with other devs clarified that the minimum Plutonium needed for the Radon loop is actually meant to be 72, not 8.

### CODE REVIEW

**Comment by boubou19** [comment]
@Steelux8 if you do the recipe of the radon loop in a LCR. Does it take 72 or 64 dusts? because if the bug is still here let's not readd a powerful plutonium dupe.

File: src/main/java/gregtech/loaders/postload/recipes/CentrifugeRecipes.java [hunk 1]
```diff
@@ -677,18 +677,33 @@ public void run() {
...
                 GT_Values.NI,
                 GT_Values.NI,
                 GT_Values.NI,
                 GT_Values.NI,
-                new int[] {2000, 200},
-                800,
-                320);
+                new int[] {1000, 500},
+                200,
+                (int) GT_Values.VP[3]);
```
Comment (boubou19) [inline src/main/java/gregtech/loaders/postload/recipes/CentrifugeRecipes.java:691]: Use the recipe enum instead, much more clear to have a `Tier.RECIPE_HV` than a `GT_Values.VP[3]`

File: src/main/java/gregtech/loaders/postload/recipes/CentrifugeRecipes.java [hunk 2]
```diff
@@ -677,18 +677,33 @@ public void run() {
...
+                GT_Values.NF,
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Uranium235, 1L),
+                GT_Values.NI,
+                GT_Values.NI,
+                GT_Values.NI,
+                GT_Values.NI,
+                GT_Values.NI,
+                new int[] {10000},
+                4000,
+                (int) GT_Values.VP[4]);
```
Comment (boubou19) [inline src/main/java/gregtech/loaders/postload/recipes/CentrifugeRecipes.java:706]: Use the recipe enum instead, much more clear to have a `Tier.RECIPE_EV` than a `GT_Values.VP[4]`

**Review by boubou19 — CHANGES_REQUESTED** [review]

File: src/main/java/gregtech/loaders/postload/recipes/CentrifugeRecipes.java (hunk 1 above)
Comment (BlueWeabo) [reply src/main/java/gregtech/loaders/postload/recipes/CentrifugeRecipes.java:691]: It's TierEU.RECIPE_HV

Comment (boubou19) [reply src/main/java/gregtech/loaders/postload/recipes/CentrifugeRecipes.java:691]: forgot Colen updated it

Comment (BlueWeabo) [reply src/main/java/gregtech/loaders/postload/recipes/CentrifugeRecipes.java:691]: Yeah an Colen's PR ain't merged for it lol iirc

**Comment by Steelux8** [comment]
> if you do the recipe of the radon loop in a LCR. Does it take 72 or 64 dusts? because if the bug is still here let's not readd a powerful plutonium dupe.

Input and output are both 64.

File: src/main/java/gregtech/loaders/postload/recipes/CentrifugeRecipes.java (hunk 1 above)
Comment (Steelux8) [reply src/main/java/gregtech/loaders/postload/recipes/CentrifugeRecipes.java]: It's there now.

File: src/main/java/gregtech/loaders/postload/recipes/CentrifugeRecipes.java (hunk 2 above)
Comment (Steelux8) [reply src/main/java/gregtech/loaders/postload/recipes/CentrifugeRecipes.java]: It's there now.

**Review by boubou19 — APPROVED** [review]

**Pull request merged**